util/settings/settings.go

```diff
@@ -367,6 +367,8 @@ const (
 	resourceExclusionsKey = "resource.exclusions"
 	// resourceInclusions is the key to the list of explicitly watched resources
 	resourceInclusionsKey = "resource.inclusions"
+	// resourceCustomLabelKey is the key to a custom label to show in node info, if present
+	resourceCustomLabelKey = "resource.customLabel"
```

Comment:
Should we support multiple labels? E.g. rename `resource.customLabel` to `resource.customLabels` and expect comma separated list of labels?

Reply:
yes, we can do that, and it's better if we do it now rather than later

Reply:
thank you @alexef ! 

> better if we do it now rather than later

yeah, it is easier to support it now than migrate later.